client/state/comments/reducer.js

```diff
@@ -319,6 +320,23 @@ export const treesInitialized = keyedReducer(
 	keyedReducer( 'status', treesInitializedReducer )
 );
 
+/***
+ * Stores the active reply comment for a given siteId and postId
+ * @param {Object} state redux state
+ * @param {Object} action redux action
+ * @returns {Object} new redux state
+ */
+export const activeReplies = createReducer(
+	{},
+	{
+		[ COMMENTS_SET_ACTIVE_REPLY ]: ( state, action ) => {
+			const { siteId, postId, commentId } = action.payload;
+			const stateKey = getStateKey( siteId, postId );
+			return { ...state, [ stateKey ]: commentId };
```

Comment:
if the commentID is null should we remove the key from the state map?

Reply:
Good idea 👍

Reply:
Added in dd396b6.